### ISSUE TITLE
Edit Membership not available for sealed topics

### ISSUE THREAD

**Issue by zaydogan** [balzack/databag#124]
When you create a sealed topic and invite multiple members, you can't edit the membership list (as the host) afterwards, which means you can't add additional members or readd members that accidentily left the group.
This is possible with unsealed topics.
I removed the !state.sealed argument in details.jsx to test and it works fine, as long everybody has setup a sealing password and unlocked it.

**Comment by zaydogan** [comment]
I stand corrected.
Adding new members is not working, they can't decrypt the messages.
Readding members that were formerly part of the group, works.

**Comment by balzack** [comment]
Yes, I am aware of this issue. A proper solution is a bit more involved, but will be included in this SDK refactor. If this is a higher priority issue, I can probably put a short term fix in until the proper solution is in place. Let me know your thoughts on the priority.....thanks for your translation btw!

**Comment by zaydogan** [comment]
it is not urgend, but it should be addressed
